addons/web/static/tests/setup.js

```diff
@@ -370,10 +380,10 @@ export async function setupTests() {
         });
         for (const node of nodes) {
             const src = node.getAttribute("src");
-            if (src && src !== "about:blank") {
+            if (src && src !== "about:blank" && src !== "/web/image/web.image_placeholder") {
```

Comment:
Same remark, is the condition actually needed here? It's better to have data-src too

Reply:
If the `src` was already edited, you don't want to override the `dataset.src` to the placeholder since the tests are testing it the for the correct value.

Reply:
Yes but this can be checked with the presence of `data-src` rather than the value of `src` (which could be the placeholder in the first place, in which case it won't have `data-src` at all).